### DISCUSSION TITLE
Loading joblib file in pyscript

### DISCUSSION THREAD

**Discussion by treeln** [pyscript/pyscript#1575] [Q&A]
Hi,

I have a trained random forest model stored in a joblib file. I am trying to load the model and make a simple static predictor web application using pyscript. But, when I run joblib.load(), I keep getting file not found error (f: FileNotFoundError: [Errno 44] No such file or directory:). I have tried using both relative and absolute paths too. Any idea how to fix this?

**Comment by JeffersGlass** [comment]
Are you including the files in a [`fetch` config in your py-config tag](https://docs.pyscript.net/latest/reference/elements/py-config.html#local-modules)? That’s the PyScript way of moving files from a URL (or your local file system exposed by a small sever program) to the virtual filesystem that python operates in.
